includes/Admin/Settings_Screens/Commerce.php

```diff
@@ -83,7 +84,29 @@ public function render_google_product_category_field() {
 	 */
 	public function get_connect_url() {
 
-		return '';
+		// build the site URL to which the user will ultimately return
+		$site_url = add_query_arg( [
+			'wc-api'               => Connection_Handler::ACTION_CONNECT_COMMERCE,
+			'nonce'                => wp_create_nonce( Connection_Handler::ACTION_CONNECT_COMMERCE ),
+		], home_url( '/' ) );
+
+		// build the proxy app URL where the user will land after onboarding, to be redirected to the site URL
+		$redirect_url = add_query_arg( 'site_url', $site_url, facebook_for_woocommerce()->get_connection_handler()->get_proxy_url() );
```

Comment:
I think we should encode `$site_url` as well:

````suggestion
		$redirect_url = add_query_arg( 'site_url', urlencode( $site_url ), facebook_for_woocommerce()->get_connection_handler()->get_proxy_url() );
```